packages/framework/aqueduct/src/container-runtime-factories/baseContainerRuntimeFactory.ts

```diff
@@ -84,6 +84,30 @@ export class BaseContainerRuntimeFactory
             existing,
         );
 
+        // const runtime: ContainerRuntime = await ContainerRuntime.newLoad(
```

Comment:
This is how we would replace the `ContainerRuntime.load()` call above with a call to `newLoad()` (final name TBD).

Reply:
@ChumpChief this is what i really want to clean up based on the conversation below. we should aim to make this as simple as possible

Reply:
- i don't like the `undefined` before entrypoint
- I don't like that the caller passes `existing`, it already exists on context, so i don't see the need to pass again, we should clean up the interfaces eventually, but i don't think the implementation needs it. it seems like a holdover from before we had the detached flow.
- runtime options feels like something we would want. i think the above undefined is the constructor. i wonder if this would be better as part of runtime options. if we move away from exposing the class, and static load, then runtime options feels like the right place. @Abe27342 i wonder if we should make this change for the existing implementation before release
- registry entries feel out of place as a param. its essential the application controls which datastores can be created, but i get the argument the entrypoint is external facing.
- uggggghhhhh.... scope... we don't have a better solution today, but would also love a better pattern here.

What if we made everything but context a single object:
```
{
   entryPoint,
  registry,
  runtimeOptions?,
  scope?,
}
``

Reply:
Ah, just saw this comment after I wrote [this](https://github.com/microsoft/FluidFramework/pull/12487#discussion_r1082953526) which I think is the same idea? I can make the change to bundle all the existing pieces into a new interface and pass that to newLoad().

Reply:
> I don't like that the caller passes existing, it already exists on context, so i don't see the need to pass again, we should clean up the interfaces eventually, but i don't think the implementation needs it. it seems like a holdover from before we had the detached flow.

It was deprecated in context: https://dev.azure.com/fluidframework/internal/_workitems/edit/994.

And poking around from there I noticed that there was already a [change](https://github.com/microsoft/FluidFramework/pull/12434/files#diff-443af31db5480232608993c92202b586dc7c32928289d66c9ac47905c8ceadb5) in `next` back in Nov around `ContainerRuntime.load()` -> `ContainerRuntime.loadRuntime()` that consolidates the parameters (context included) which should go out in the major release next week.

So I think I'll hold further changes on this PR until next gets merged back into main and then work on top of that version to add the entryPoint initialization to the `params` object.